doc/source/myrocks/variables.rst

```diff
@@ -2159,7 +2159,7 @@ Disabled by default.
   :vartype: Numeric
   :default: ``1``
 
-Specifies whether to verify that MySQL ``.frm`` files match MyRocks tables.
+Specifies whether to verify that MySQL data dictionary is equal to the MyRocks data dictionary.
```

Comment:
`rocksdb_validate_tables` should have separate description for 5.7 and 8.0:
In 5.7 we still have .FRM
In 8.0 we have data dictionary